handlers/secrets_api_test.go

```diff
@@ -198,3 +197,71 @@ func Test_SecretsHandler_ListEmpty(t *testing.T) {
 		t.Errorf(`want empty list to be valid json i.e. "[]", but was %q`, string(body))
 	}
 }
+
+func Test_GetLookupNamespace(t *testing.T) {
+	defaultNamespace := "openfaas-fn"
```

Comment:
Would this work as a test-case table instead? Example: https://blog.alexellis.io/golang-writing-unit-tests/

Reply:
I can look into refactoring into table tests, let's just get everything right first 💃